sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -36,6 +36,18 @@ export interface AnalyzeResult {
 // @public
 export type AnalyzeTextOptions = OperationOptions & AnalyzeRequest;
 
+// @public
+export interface AnswerResult {
+    [property: string]: any;
+    readonly highlights?: string | null;
+    readonly key?: string;
+    readonly score?: number;
+    readonly text?: string;
```

Comment:
is everything in the result actually optional? Feels weird that this interface is essentially "an object, with maybe some properties, but also any other properties"

Reply:
Yes. It looks a little odd. I will confirm in today's meeting and update this PR.

Reply:
Fixed it